### PR TITLE
feat(infotips): Added info tips in planner

### PR DESCRIPTION
**Pull request by divyanshiGupta**
Info tips User Story: https://openshift.io/openshiftio/openshiftio/plan/detail/1744
Test: WIP

Notes:
- Currently, info tips are hardcoded in the frontend as the backend is not ready, once the backend is ready hardcoded info tips will be removed.
- For few UI elements, info tips were not present in the JSON doc so I have used lorem ipsum text in its place

### CODE REVIEW

**Comment by alien-ike** [comment]
### Ike Plugins (test-keeper)

Thank you @divyanshiGupta for this contribution!



It appears that no tests have been added or updated in this PR.

Automated tests give us confidence in shipping reliable software. Please add some as part of this change.

If you are an admin or the reviewer of this PR and you are sure that no test is needed then you can use the command `/ok-without-tests` as a comment to make the status green.


For more information please head over to official [documentation](http://arquillian.org/ike-prow-plugins/#_test_keeper_plugin). You can find there how to configure the plugin - for example exclude certain file types so if PR contains only them it won't be checked.

File: src/app/components_ngrx/infotip/infotip.component.ts [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  template: `<template #popoverTemplate>
+             <span>{{infotipText}}</span>
+             <span class="pficon-close close-popover" (click)="handlePropagation($event); pop.hide();"></span>
+             </template>
+             <span class="pficon-info infotip-icon"
+             (click)="handlePropagation($event)" 
+             [popover]="popoverTemplate" placement="right" 
+             containerClass="popover-container" #pop="bs-popover">
+             </span>`
+})
```
Comment (vikram-raj) [inline src/app/components_ngrx/infotip/infotip.component.ts]: Create a separate file for the template.

File: src/app/components_ngrx/group-types-panel/group-types-panel.component.ts [hunk 2]
```diff
@@ -24,7 +24,7 @@ import * as GroupTypeActions from './../../actions/group-type.actions';
   styleUrls: ['./group-types-panel.component.less']
 })
 export class GroupTypesComponent implements OnInit, OnDestroy {
-
+
```
Comment (vikram-raj) [inline src/app/components_ngrx/group-types-panel/group-types-panel.component.ts]: remove this extra line space.

File: src/app/components_ngrx/group-types-panel/group-types-panel.component.less [hunk 3]
```diff
@@ -25,6 +25,7 @@
     &:visited, &:focus {
       color: @color-pf-white;
       text-decoration: none;
+
```
Comment (vikram-raj) [inline src/app/components_ngrx/group-types-panel/group-types-panel.component.less]: remove this extra new line.

File: src/app/components_ngrx/group-types-panel/group-types-panel.component.html [hunk 4]
```diff
@@ -1,5 +1,5 @@
 <div class="f8-group-filter" *ngFor="let item of groupTypes">
-  <a
+  <a
```
Comment (vikram-raj) [inline src/app/components_ngrx/group-types-panel/group-types-panel.component.html]: remove this extra space.

**Comment by rgarg1** [comment]
@divyanshiGupta Build failed

**Comment by divyanshiGupta** [comment]
@Raunak1203 done

**Comment by divyanshiGupta** [comment]
@Raunak1203 according to UX design the infotip closes only when clicked on the cross button not on outside click. I will check with them again

**Pull request merged**